README.md

```diff
@@ -928,6 +942,19 @@ console.log(prettyDOM(div))
 
 This function is what also powers [the automatic debugging output described above](#debugging).
 
+## Configuration
+
+The library can be configured via the `configure` function, which accepts:
+
+- a plain JS object; this will be merged into the existing configuration. e.g. `configure({testIdAttribute: 'not-data-testid'})`
+- a function; the function will be given the existing configuration, and should return a plain JS object which will be merged as above, e.g.
+  `configure(existingConfig => ({something: [...existingConfig.something, 'extra value for the something array']}))`
+
+Configuration options:
+
+`testIdAttribute`
+: The attribute used by `getByTestId` and related queries. Defaults to `data-testid`. See [`getByTestId`](#getbytestid).
```

Comment:
Can this be placed on the same line so there's no space in the rendered output?

<img width="171" alt="screen shot 2018-11-29 at 8 40 37 am" src="https://user-images.githubusercontent.com/1500684/49237131-7657e480-f3b2-11e8-9250-d3892cf133ae.png">

Reply:
Hmm. Might be another `prettier` funsie. I'm sure I put them on the same line originally. I'll have a go...